azure-pipelines.yml

```diff
@@ -48,26 +49,12 @@ jobs:
   - task: CopyFiles@2
     inputs:
       sourceFolder: '$(Build.SourcesDirectory)'
-      contents: '*.tgz' 
+      contents: '*.tgz'
       targetFolder: $(Build.ArtifactStagingDirectory)/npm
     displayName: 'Copy npm package'
 
-  - task: CopyFiles@2
```

Comment:
No need for these files? Do you know why this was here? Unrelated to the other PR, just cleanup?

Reply:
The folder structure was slightly different. Basically package-lock was manually copied over, but instead I made sure to add it to the package created by `npm pack`, so we take it from there. So yeah, it's related to the other PR (already merged).